website/project/model.py

```diff
@@ -397,10 +397,19 @@ class NodeLog(StoredObject):
     REGISTRATION_APPROVAL_INITIATED = 'registration_initiated'
     REGISTRATION_APPROVAL_APPROVED = 'registration_approved'
 
+    @classmethod
+    def actions(cls):
```

Comment:
Minor: Is there any reason this shouldn't be a class variable? Also, better to use a set for constant-time membership lookup.

```python
class NodeLog(StoredObject):
    #...
    ALL_ACTIONS = set([
        CREATED_FROM, PROJECT_CREATED, ...
    ])
```